src/Stratis.Bitcoin/Consensus/CoinViews/DBreezeCoinView.cs

```diff
@@ -34,17 +34,17 @@ public class DBreezeCoinView : ICoinView, IDisposable
         /// <summary>Hash of the block which is currently the tip of the coinview.</summary>
         private uint256 blockHash;
 
-        /// <summary>Performance counter to measure performance of the database insert and query operations.</summary>
-        private readonly BackendPerformanceCounter performanceCounter;
+        private const string RewindDataKey = "RewindData";
+
+        private const string CoinsKey = "Coins";
+
+        private const string BlockHashKey = "BlockHash";
 
         /// <summary>Performance counter to measure performance of the database insert and query operations.</summary>
-        public BackendPerformanceCounter PerformanceCounter
-        {
-            get { return this.performanceCounter; }
-        }
+        public BackendPerformanceCounter PerformanceCounter { get; }
 
         /// <summary>Provider of time functions.</summary>
-        protected readonly IDateTimeProvider dateTimeProvider;
+        protected readonly IDateTimeProvider DateTimeProvider;
```

Comment:
might be private

Reply:
Removed it in favour of a local variable.